server/utils/nunjucksSetup.ts

```diff
@@ -74,6 +84,31 @@ module.exports = (app, path) => {
     }))
   })
 
+  njkEnv.addFilter('toPagination', (pageData: PageMetaData) => {
```

Comment:
adding function to allow mapping from PageMetadata to model required by nunjucks template